src/renderer/main/panels/configuration/Configuration.store.js

```diff
@@ -426,7 +426,9 @@ export class ConfigTarget {
 export const configManager = create_configuration_manager();
 
 function create_configuration_manager() {
-  let store = writable(new ConfigList());
+  let store = derived(user_input, (ui) => {
```

Comment:
You can't simply slap a `derived()` store in place of a `writable()` store and expect it to work. 

By default, a derived store's value is updated based on the input store's values, first argument of the `derived()` svelte store.

**Please revert this code change.** And return to it's discussion at a future date.

The below update came up the moment I tried to apply a change within Editor with Grid module connected. Did you try your code change before push?

<img width="1664" alt="image" src="https://github.com/intechstudio/grid-editor/assets/47832952/ffbcce3f-bfe8-4cc1-b519-c9694226db59">